src/css-grids/cssgrids.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CSS-Grids Dashboard</title>
+    <link rel="stylesheet" href="cssgrids-styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Orbitron' rel='stylesheet' type='text/css'>
+    <script src="https://kit.fontawesome.com/7001303531.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <main class="main-container">
+        <nav class="topnavbar">
+            <i class="fas fa-envelope"></i>
+            <i class="fas fa-tools"></i>
+            <i class="fas fa-bell"></i>
+            <i class="fas fa-user"></i>
```

Comment:
because I'm recommending using font awesome version 4.7. you'll need to change the` fas` class to just `fa`
checkout this link to find a replacement for the fa-tools class. 
[https://fontawesome.com/v4.7.0/icons/](url)